src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/NLGSpellerInterop.cs

```diff
@@ -643,6 +643,16 @@ private void EnumerateSubSegments()
 
             #region SpellerInteropBase.ISpellerSegment
 
+            /// <summary>
+            /// <inheritdoc/>
```

Comment:
The other code only uses the `/// <inheritdoc/>` without `/// <summary>` see:

https://github.com/dotnet/wpf/blob/7afb994e07336fc4bfb1c74c5c9669e0c464d2e2/src/Microsoft.DotNet.Wpf/src/Shared/System/Windows/Markup/Replacements/TypeUriConverter.cs#L21

Reply:
@lindexi thanks for the suggestions - i've incorporated them.